CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
```

Comment:
Rename to "fix starting crc via the extension"

Reply:
Acutally, use Mohit's suggestion

Reply:
Used Mohit's version for this: "Bump minimum memory for CRC to 10752 MB"